Code/Tools/ProjectManager/Source/Settings.cpp

```diff
@@ -167,15 +168,16 @@ namespace O3DE::ProjectManager
         bool success = GetBuiltSuccessfullyPaths(builtPathsResult);
 
         AZStd::string projectPath = projectInfo.m_path.toStdString().c_str();
+        AZ::IO::FixedMaxPath lexicallyNormalPath = AZ::IO::FixedMaxPath(projectPath).LexicallyNormal();
```

Comment:
Repeat: the projectInfo.m_path can be converted to a temporary QByteArray that is supplied to AZ::IO::Path, for the normalization of the path so that it can be stored.

Since the path is be inserted into into an `AZStd::set<AZ::IO::Path>`, the lexicallyNormalPath member can be moved into it.

```suggestion
        AZ::IO::Path lexicallyNormalPath = AZ::IO::Path(projectInfo.m_path.toUtf8().constData()).LexicallyNormal();
```